client/r/rhawc/DESCRIPTION

```diff
@@ -0,0 +1,20 @@
+Package: rhawc
+Type: Package
+Title: HAWC (Health assessment workspace collaborative) client 
+Version: 0.1.0
+Author: c(person("Mckayla", "Lein", email="mckayla.lein@gmail.com", role=c("cre", "cst", "dnc","spy")),
+    person("Daniel", "Rabstejnek", email="rabstejnek.daniel@epa.gov", role=c("aqt")))
+Maintainer: Mckayla Lein <mckayla.lein@gmail.com>
+Description: Doing cool things in the name of HAWC. This package is a knock off version of the Python client. 
```

Comment:
change the description to "An R-client to send and receive data from HAWC (Health assessment workspace collaborative).`